### ISSUE TITLE
blacklight-marc depends on traject ~> 2.1

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_blacklight#138]
```
Bundler could not find compatible versions for gem "traject":
  In Gemfile:
    traject (= 3.0.0)

    blacklight-marc was resolved to 7.0.0.alpha, which depends on
      traject (~> 2.1)
```

**Comment by cdmo** [comment]
🤔  Can't multiple versions of a gem exist in a project?

**Comment by banukutlu** [comment]
seem

> 🤔 Can't multiple versions of a gem exist in a project?

seems to be not 🤔  https://stackoverflow.com/questions/4333645/use-2-versions-of-gem-at-same-time

**Comment by cdmo** [comment]
>  blacklight-marc was resolved to 7.0.0.alpha

🤔 The other odd thing about the error message is that blackight-marc doesn't have a 7.0.0.alpha release/tag or branch. So... ?

**Comment by banukutlu** [comment]
@cdmo it is here: https://github.com/psu-libraries/blacklight-marc/blob/master/lib/blacklight/marc/version.rb


I was not able to test if this issue has been solved yet. I will be able to test when merged.
